pds_github_util/gh_pages/summary.py

```diff
@@ -5,16 +5,19 @@
 from pds_github_util.corral.herd import Herd
 from uuid import uuid4
 
+logger = logging.getLogger('github3')
+logger.setLevel(level=logging.WARNING)
+
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
-COLUMNS = ['manual', 'changelog', 'requirements', 'download', 'license', 'feedback']
+COLUMNS = ['Manual', 'Changelog', 'Requirements', 'Download', 'License', 'Feedback']
```

Comment:
The column names are not used to be display since we want to reduce the width with a vertical layout of the text. There are only identifier which are used later in the code.

If you want to capitalize the column title we need to regnerate the images at https://github.com/NASA-PDS/nasa-pds.github.io/tree/master/images